share/cmake/utils/CompilerFlags.cmake

```diff
@@ -12,7 +12,8 @@ if(USE_MSVC)
     set(PLATFORM_COMPILE_FLAGS "${PLATFORM_COMPILE_FLAGS} /DUSE_MSVC")
 
     # /we4062 Enables warning in switch when an enumeration value is not explicitly handled.
-    set(PLATFORM_COMPILE_FLAGS "${PLATFORM_COMPILE_FLAGS} /EHsc /DWIN32 /we4062")
+    # /we4701 Potentially uninitialized local variable 'name' used
```

Comment:
If that's needed by `fast_float`, please add a comment like `... because of fast_float vX.X.X`

Reply:
Not exactly, I added this because OCIO's current GCC/Clang setup issues `-Wall -Werror` and that detects uninitialized variables, unlike MSVC (which I work with). So I end up noticing missing bits only at CI time.

Reply:
I will add the explanation to be further clear, as this may affect only the strtod-based from_chars (which leave the value variable explicitly untouched on failure).